### ISSUE TITLE
Increment timestamp ignore 1970-01-01 00:00:00

### ISSUE THREAD

**Issue by Sano123456** [confluentinc/kafka-connect-jdbc#541]
we encounter this bug, when the column have 1970-01-01 00:00:00 the jdbc connector it ignore them

is this normal?

**Comment by aliasbadwolf** [comment]
@Sano123456 Yes, this is normal as working as expected. The Kafka Connect uses this time as the default starting time in case of any mode that uses timestamp when you are running the connect for the first time and there are no offsets in the offsets topic of the connect framework (and -1 for incrementing mode default).

This is not a bug. its the implementation of Kafka Connect.
